tensorboard/webapp/core/effects/core_effects_test.ts

```diff
@@ -85,7 +85,7 @@ describe('core_effects', () => {
           },
         },
       },
-    });
+    } as unknown) as HTMLElement);
```

Comment:
What is the rational to cast the obj to unknown?

Reply:
ah, this spy is stubbing `createElement` which is supposed to return `HTMLElement` but we are returning an object here. An object is not an instance of a `HTMLElement` but we are asking TypeScript to forget the type of the object and coerce `unknown` to `HTMLElement`.